### DISCUSSION TITLE
求助！！字符检测，只抠出文字，其余部分不需要

### DISCUSSION THREAD

**Discussion by 1hexf1** [PaddlePaddle/PaddleOCR#12556] [Q&A]
![mask_final](https://github.com/PaddlePaddle/PaddleOCR/assets/90681516/4ad39f2a-eafe-4792-854f-268f7ee8fb3d)
各位大佬，请问应该用什么算法可以达到这种效果，因为我想只提取文字，使用PIL库让我生成的文字和之前的保持相同的颜色。请问应该怎么达到这个效果

**Comment by hjj-lmx** [comment]
> ![mask_final](https://private-user-images.githubusercontent.com/90681516/290831601-4ad39f2a-eafe-4792-854f-268f7ee8fb3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4MjA3NTgsIm5iZiI6MTczNjgyMDQ1OCwicGF0aCI6Ii85MDY4MTUxNi8yOTA4MzE2MDEtNGFkMzlmMmEtZWFmZS00NzkyLTg1NGYtMjY4ZjdlZThmYjNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTE0VDAyMDczOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFjZWUyYWEzMzhiODBhOWY4NWM0NWI3YmNmYmQ5MDg5YmJkNTQ5YWI1OWVjNmMxYmIwMGJjMzQ5MDUwYWZlYjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.m3Z0jyuRkPer3_pUsY7_-0-sV_BoUf6rW4o-STVgbss) 各位大佬，请问应该用什么算法可以达到这种效果，因为我想只提取文字，使用PIL库让我生成的文字和之前的保持相同的颜色。请问应该怎么达到这个效果

请问你实现了这种效果我，我也想只要图片中文字的mask

**Reply by h123fire** [reply]
轮廓提取？
